openpype/hosts/maya/plugins/publish/validate_path_for_plugin.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
Should we maybe rename this plugin to `validate_path_attributes`?

Whether its a native node in Maya or not, the path attributes still need to be correct. We can include any native Maya files in the plugin itself and leave the plugin nodes/attributes for the settings.